site/src/github.rs

```diff
@@ -700,7 +700,25 @@ async fn categorize_benchmark(
             primary.unwrap_or_else(|| ComparisonSummary::empty()),
             secondary.unwrap_or_else(|| ComparisonSummary::empty()),
         );
-        write_summary_table(&primary, &secondary, &mut result);
+        write_summary_table(&primary, &secondary, true, &mut result);
+
+        if !primary.errors_in().is_empty() || !secondary.errors_in().is_empty() {
+            let list_errored_benchmarks = |summary: ComparisonSummary| {
+                summary
+                    .errors_in()
+                    .iter()
+                    .map(|benchmark| format!("- {benchmark}"))
+                    .collect::<Vec<_>>()
+                    .join("\n")
+            };
+            write!(
+                result,
+                "\nThe following benchmark(s) failed to build:\n{}{}\n",
+                list_errored_benchmarks(primary),
```

Comment:
We currently duplicate the same errors into `primary` and `secondary` (see `summarize_by_category`). This would result in each error being printed twice.

So either this logic should be reverted or we should properly split the errors by their category. TBH I don't think that we need to distinguish build errors by category, so I would just revert this change.